docs/source/transaction_family_specifications/supplychain_transaction_family.rst

```diff
@@ -0,0 +1,255 @@
+    }
+
+
+Agent Transactions
+------------------
+
+Create Agent
+++++++++++++
+
+Create an agent record, agent address will be calculated from the public key
+extracted from the transactions signature.
+
+.. code-block:: javascript
+
+    {
+        MessageType: 'Agent',
+        Action: 'Create',
+        // Create Agent fields
+        Name: 'name',  // Name of the agent to create
+        Type: 'type',  // Domain relevant type
+        Url: 'url',  // Link for more information
+    }
+
+Record Operations
+-----------------
+
+Create Record
++++++++++++++
+
+Create a record of an item to be tracked. The current Owner and Custodian is
+set to the signer of this transaction.
+
+.. code-block:: javascript
+
+    {
+        MessageType: 'Record',
+        Action: 'Create',
+        // Create Record fields
+        RecordId: record_addr, // address
+        Timestamp: timestamp // current time, seconds from the epoch
+    }
+
+Create Application
+++++++++++++++++++
+
+Create an application for transfer of ownership or custodianship of the record.
+
+.. code-block:: javascript
+
+    {
+        MessageType: 'Record',
+        Action: 'CreateApplication',
+        // Create Application fields
+        RecordId: record_address,
+        ApplicationType: "Owner" | "Custodian",
+        ApplicationTerms: 'terms', // human readable terms of the transfer.
+    }
+
+Accept Application
+++++++++++++++++++
+
+Accept an application for transfer of ownership or custodianship of the record. Must be submitted by the current owner or custodian.
+
+.. code-block:: javascript
+
+    {
+        MessageType: 'Record',
+        Action: 'AcceptApplication',
+        // Accept Application fields
+        RecordId: record_address
+    }
+
+Reject Application
+++++++++++++++++++
+
+Reject an application for transfer of ownership or custodianship of the record. Must be submitted by the current owner or custodian.
+
+.. code-block:: javascript
+
+    {
+        MessageType: 'Record',
+        Action: 'RejectApplication',
+        // Reject Application Fields
+        RecordId: record_address,
+    }
+
+Cancel Application
+++++++++++++++++++
+
+Cancel an application for transfer of ownership or custodianship of the record. Must be submitted by the Applicant.
+
+.. code-block:: javascript
+
+    {
+        MessageType: 'Record',
+        Action: 'CancelApplication',
+        // Cancel Application Fields
+        RecordId: record_address,
+    }
+
+Finalize Record
++++++++++++++++
+
+Mark the record as final (no longer able to be updated). The owner must be
+the current custodian and this transaction must be signed by the owner.
+
+.. code-block:: javascript
+
+    {
+        MessageType: 'Record',
+        Action: 'Finalize',
+        // Finalize Record Fields
+        RecordId: record_address
+    }
+
+Execution
+=========
+
+The current implementation simple and does not have many validity checks in
```

Comment:
missing verb "is":
s/The current implementation simple and/The current implementation is simple and/